specification/agrifood/resource-manager/Microsoft.AgFoodPlatform/preview/2023-06-01-preview/agfood.json

```diff
@@ -0,0 +1,2991 @@
+{
```

Comment:
Given the amount of changes, do you want the reviewer to review whole swagger and provide feedback? It is difficult to provide feedback for these many changes by doing diff.

Reply:
@visingla-ms we can have a call today to discuss the changes. Is tha fine?